src/components/PreviewCard/PreviewCard.tsx

```diff
@@ -64,6 +76,28 @@ function PreviewCard({
           </View>
         </View>
         <View style={styles.tagsContainer}>
+          <View>
+            <TouchableOpacity
```

Comment:
You can remove the touchable opacity here and move the styles to the outer view. We want the user to go to the story once they click the reactions, which should be handled by the outer touchable opacity